feature-tests/run-tests.sh

```diff
@@ -60,6 +60,7 @@ test_run_result=$?
 
 # store report to dir where pipeline will export from
 reportDir=${TEST_REPORT_ABSOLUTE_DIR:-/results}
+echo "Report directory is set to: $reportDir"
```

Comment:
Can we also add 
echo "Results folder content : $(ls -altr results)"

Reply:
Thanks Jamie, PR updated